test/fheap.c

```diff
@@ -16043,7 +16043,7 @@ main(void)
                 if (H5Pset_file_space_strategy(fcpl, H5F_FSPACE_STRATEGY_FSM_AGGR, true, (hsize_t)1) < 0)
                     TEST_ERROR;
                 fapl = def_fapl;
-                /* This is a fix for the daily test failure from the checkin for libver bounds. */
```

Comment:
Use "commit" instead of "checking" so codespell doesn't try to suggest "checking"